modules/utxo-lib/src/bitgo/legacysafe/index.ts

```diff
@@ -48,8 +48,9 @@ export function createLegacySafeOutputScript2of3(
   }
 
   pubkeys.forEach((key) => {
-    if (key.length !== 65) {
-      throw new Error(`Unexpected key length ${key.length}. Must use uncompressed keys.`);
+    // V1 Safe BTC wallets could contain both uncompressed and compressed pubkeys
```

Comment:
technically I believe they must either all be compressed or uncompressed but we probably won't encounter a mixed-length scenario